src/WebdriverClassicDriver.php

```diff
@@ -937,15 +937,24 @@ private function applyTimeouts(): void
                     case 'script':
                         $timeouts->setScriptTimeout($param / 1000);
                         break;
+
                     case 'implicit':
                         $timeouts->implicitlyWait($param / 1000);
                         break;
+
                     case 'page load':
                     case 'pageLoad':
-                        // ☝️ For MinkSelenium2Driver backward compatibility
+                        @trigger_error(
+                            "Using \"$type\" timeout type is deprecated, please use \"page\" instead",
+                            E_USER_DEPRECATED
+                        );
+                        $timeouts->pageLoadTimeout($param / 1000);
+                        break;
```

Comment:
This part isn't needed.

Reply:
I know, but I get IDE warnings that it might be a mistake ("that I forgot to break;"). 🙈

Reply:
Replace that code (highlighted by this comment) with `// no break` line. It should work in PhpStorm (if you're using it) according to the https://youtrack.jetbrains.com/issue/WI-32123/Usage-of-PSR-2-comment-suggestion-on-no-break-switch-cases .